Geometry/CommonDetUnit/interface/GluedGeomDet.h

```diff
@@ -0,0 +1,28 @@
+#ifndef Geometry_TrackerGeometryBuilder_GluedGeomDet_H
+#define Geometry_TrackerGeometryBuilder_GluedGeomDet_H
```

Comment:
@slava77  - Thanks for pointing it out. I'll fix it in the next PR along with cleaning up spurious dependencies on Geometry package.